version.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/urfave/cli"
+)
+
+// version will be populated by the Makefile, read from
+// VERSION file of the source code.
+var version = ""
+
+// gitCommit will be the hash that the binary was built from
+// and will be populated by the Makefile
+var gitCommit = ""
+
+var versionCommand = cli.Command{
+	Name:        "version",
+	Usage:       "output the runtime version",
+	Description: `The version command outputs the runtime version.`,
+	Action:      printVersion,
+}
+
+func printVersion(context *cli.Context) (err error) {
```

Comment:
> No where else in our code do we use these functions on os.Stdout…

I'm replacing the `os.Stdout.WriteString` calls with `fmt.Print*` calls.

> … and all the branches with if/else are just bad.

Most of the `if` blocks are for error checking, and I don't know how to avoid that in Go.  The only `else` is to cover “have/don't-have a version?”, and I think we want to cover both sides of that.  Besides error checking and version presence, the only other conditional is for “have a Git commit?”, and I think we want to cover that too.  Do you have any conditionals you'd like to see dropped?